### PR TITLE
Add client to node.

### PR DESCRIPTION
**Pull request by dvc94ch**
Superceeds #51

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,12 +1,55 @@
...
+    "node",
+    "modules/util",
+    "modules/shares-atomic",
+    "modules/vote-yesno",
+    "modules/bank",
+    "runtime",
 ]
+
+[patch.crates-io]
+#frame-executive = { git = "https://github.com/paritytech/substrate.git", rev = "8c1858048148c1c717123b0e4d360f8ad1be5bf2" }
```
Comment (4meta5) [inline Cargo.toml:16]: why are we keeping these? It seems .6 was released and you used that everywhere instead right?

Comment (dvc94ch) [reply Cargo.toml:16]: In case we have to patch again, but I can remove it...

File: client/src/frame/vote_yesno.rs [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+/// The organization identifier type
+pub type OrgId<T> =
+    <<T as VoteYesNo>::ShareData as ShareRegistration<<T as System>::AccountId>>::OrgId;
+
+const MODULE: &str = "VoteYesNo";
+const CREATE_SHARE_WEIGHTED_PERCENTAGE_VOTE_NO_EXPIRY: &str =
+    "create_share_weighted_percentage_threshold_vote";
+const CREATE_SHARE_WEIGHTED_COUNT_VOTE_NO_EXPIRY: &str =
+    "create_share_weighted_count_threshold_vote";
+const CREATE_1P1V_PERCENTAGE_VOTE_NO_EXPIRY: &str = "create_1p1v_share_weighted_threshold_vote";
```
Comment (4meta5) [inline client/src/frame/vote_yesno.rs:60]:
```suggestion
const CREATE_1P1V_PERCENTAGE_VOTE_NO_EXPIRY: &str = "create_1p1v_percentage_threshold_vote";
```

Comment (4meta5) [reply client/src/frame/vote_yesno.rs:60]: I don't think that the most recent commit from the other branch is included so I won't commit this suggestion until I know the method that corresponds to it is in this PR (and it isn't now based on the code shown in vscode)

File: Cargo.toml (hunk 1 above)
Comment (4meta5) [reply Cargo.toml:16]: I see, we can keep it. We probably will have to patch again at some point

**Comment by 4meta5** [comment]
This method needs to be added to the `decl_module` of `modules/vote-yesno` as per the previous comment.

```rust
#[weight = SimpleDispatchInfo::zero()]
pub fn create_1p1v_percentage_threshold_vote(
    origin,
    organization: OrgId<T>,
    share_id: ShareId<T>,
    support_requirement: Permill,
    turnout_requirement: Permill,
) -> DispatchResult {
    let vote_creator = ensure_signed(origin)?;
    // default authentication is organization supervisor or sudo key
    let authentication: bool = <<T as Trait>::ShareData as OrgSupervisorKeyManagement<
        OrgId<T>,
        <T as frame_system::Trait>::AccountId,
    >>::is_organization_supervisor(organization, &vote_creator) ||
    <<T as Trait>::ShareData as SudoKeyManagement<
        <T as frame_system::Trait>::AccountId,
    >>::is_sudo_key(&vote_creator);
    ensure!(authentication, Error::<T>::NotAuthorizedToCreateVoteForOrganization);
    let threshold_config = ThresholdConfig::new_percentage_threshold(support_requirement, turnout_requirement);
    // share weighted count threshold
    let new_vote_id = Self::open_vote(organization, share_id, None, SupportedVoteTypes::ShareWeighted, threshold_config, None)?;
    // emit event
    Self::deposit_event(RawEvent::NewVoteStarted(organization, share_id, new_vote_id));
    Ok(())
}
```

I was going to add it to the fork but the fork is on your repo and I just suck at git

**Pull request merged**